custom_button/CustomButton.py

```diff
@@ -0,0 +1,101 @@
+import tkinter as tk
+from tkinter import *
+from external.constants import FONT_FAMILY, FONT_SIZE
+
+class CustomButton:
+    """Class, whick represents custom button."""
+
+    color = "#21A2F1"
+    activebackground = "white"
+    width = 70
+    height = 40
+    index = 0
+    """Static arguments which represent button
```

Comment:
Nice idea, but it doesn't work as you expected. This whole text is assigned only last variable `index`. So you need to document each variable separately (you can see it, when you try to use some of them in another file)